pages/cloud-solutions.md

```diff
@@ -14,3 +14,23 @@ link_cards:
     permalink: /implementation/
 ---
 
+Cloud offerings are usually organized in one of two ways: either by deployment model (what the product is) or business need (what the product does). 
+
+Government procurement is typically organized around deployment model rather than business need. Doing so allows government to both align with [NIST guidance](https://csrc.nist.gov/publications/detail/sp/800-145/final) and to offer simplified, cost-effective acquisition via [blanket purchase agreements](https://www.gsa.gov/technology/technology-purchasing-programs/cloud-acquisition-vehicles). Industry, on the other hand, tends to organize its offerings around what a cloud solution can help you accomplish.
+
+Although you’ll ultimately choose an IaaS, PaaS, or SaaS solution, start by considering your business need rather than your deployment model. There may be products that accomplish your needs with more automation or less compliance work than you anticipate.
+
+The following cloud products are arranged by business need. Each of them is [FedRAMP-authorized](https://www.fedramp.gov/about/), meaning they’re easier and cheaper to acquire and deploy. 
+
+## Application development
```

Comment:
@jdamis4XD can you link these to the internal page? or should they be presented as cards?